.github/workflows/release-versoview.yml

```
@@ -124,17 +129,17 @@ jobs:
         env:
           SCCACHE_GHA_ENABLED: true
           RUSTC_WRAPPER: 'sccache'
-        run: cargo build --release
+        run: cargo build --target ${{ matrix.target }} --release
 
       - name: Make archive
         working-directory: ./target/release
```

I think you'll also need this, cargo puts the artifacts in a separate directory when using `--target`

```suggestion
        working-directory: ./target/${{ matrix.target }}/release
```